### PR TITLE
redesigned the navbar

### PR DESCRIPTION
**Pull request by Izcomayana**
Implemented the navabar dropdown on dashboard
Made the profile drop-down responsive on the dashboard.

### CODE REVIEW

File: src/components/CreateEvent/CreateEventNavbar.jsx [hunk 1]
```diff
@@ -39,40 +34,45 @@ const CreateEventNavbar = () => {
...
+      <div class="text-left">
+        {open && 
+          <div class="absolute right-6 z-10 w-56 origin-top-right divide-y divide-gray-100 rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="menu-button" tabindex="-1">
+            <div class={`${menuStyles.goTo} py-3 px-2`} role="none">
+              <ul>
+                <li>
+                  <BsPerson />
+                  <Link to="/dashboard/upcoming_events" className="block px-2 py-2 text-sm">
+                    Profile
+                  </Link>
```
Comment (vickymarz) [inline src/components/CreateEvent/CreateEventNavbar.jsx:46]: - [ ] Kindly use the correct link for the profile

Here is the link 👇🏽 
`<Route path="/profile" element={<ProfilePage />} />`

File: src/components/CreateEvent/CreateEventNavbar.jsx [hunk 2]
```diff
@@ -39,40 +34,45 @@ const CreateEventNavbar = () => {
...
+                </li>
+                <li>
+                  <BiPurchaseTag />
+                  <Link to="/#" className="block px-2 py-2 text-sm">
+                    Faq
+                  </Link>
+                </li>
```
Comment (vickymarz) [inline src/components/CreateEvent/CreateEventNavbar.jsx:53]: - [ ] Kindly add FAQ routes

**Review by vickymarz — CHANGES_REQUESTED** [review]
Hi @Izcomayana, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 



## Required Changes ♻️

- [ ] I noticed that when i am reducing the screen size to mobile view, the navbar becomes visible which is not a good user experience. KIndly fix it

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/CreateEvent/CreateEventNavbar.jsx (hunk 2 above)
Comment (Izcomayana) [reply src/components/CreateEvent/CreateEventNavbar.jsx:53]: I don't know the faq route

**Comment by vickymarz** [comment]
@Izcomayana check the app.jsx component

File: src/components/CreateEvent/CreateEventNavbar.jsx (hunk 1 above)
Comment (Izcomayana) [reply src/components/CreateEvent/CreateEventNavbar.jsx:46]: Okay

**Review by vickymarz — APPROVED** [review]
Hi @Izcomayana, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Navbar working properly ✅ 
- Navbar is responsive✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**